.gitignore

```diff
@@ -5,3 +5,4 @@ out
 third_party/mongodb
 third_party/node
 database
+.babel-cache.json
```

Comment:
We should keep a alphabetical order.
So, this change should be at line number 2. (before .pkg_timestamp)